CancerRelatedProcedures.fsh

```diff
@@ -35,7 +32,7 @@ Conformance note: If an ICD-10-PCS code is used in the code attribute, and there
 * recorder only Reference(Practitioner | PractitionerRole)
 * performer.actor only Reference(Practitioner | PractitionerRole | Organization)  // include Device?
 * reasonCode from CancerDisorderVS (extensible)
-* reasonReference only Reference(PrimaryCancerCondition | SecondaryCancerCondition)
+* reasonReference only Reference(CancerConditionParent)
```

Comment:
I thought CancerConditionParent is an "abstract class".  Specifying it this way from an implementation perspective, implies that we should now be exposing `CancerConditionParent` in the IG and explicitly referenced in our FHIR instances.  This differs significantly from the previous IG. Makes sense to keep it the way it was before, explicitly calling out PrimaryCancerCondition and SecondaryCancerCondition since this is what's been recognized by the mCODE TRG in the conceptual model.

Reply:
We discussed this. It is a better approach in terms of extensibility. It is really because of not wanting to define `Tumor` which was a hidden SD in the ballot version. This allows Tumor (or another kind of cancer condition, to be defined later without needing to change mCODE.